index.html

```diff
@@ -31,48 +50,79 @@ <h3>"Experience Professional-grade Sound: Record Your Next Hit with us."</h3>
       </div>
       <div class="visit-time">
       <h2>
-        Visit us any time: MON-SAT 8h00-18h00.
+        Visit us any time from MON-SAT between 8h00-18h00.
       </h2>
     </div>
     </div>
-      <div class="services">
+      <div class="services" id="services">
```

Comment:
Great design @God-Logic 🤝

- [ ] Kindly ensure this section consists of 5 cards instead of 6 cards. We are trying to replicate the design as much as possible.
- Your design.
![cards](https://user-images.githubusercontent.com/63917725/233805747-61d08e6f-6d3e-4480-b24c-1bfbc6a48e88.PNG)
- Given design.
![given cards](https://user-images.githubusercontent.com/63917725/233805777-d055058a-0a02-4c20-a32c-7a66e3fec1fa.PNG)